### PR TITLE
resolved #492 Add speakers: Sam Witteveen, Jan Peuker, Allen Day, Kin Ko, Vlado Koljibabic

### LINKED ISSUE #492: Add speakers: Sam Witteveen, Jan Peuker, Allen Day, Kin Ko, Vlado Koljibabic
please use their info from here https://eventyay.com/e/b548bcfc/#speakers
and use their photos below:

Kin Ko
9h3a3730 1

Sam Witteveen

Jan Peuker (jus use this one, couldn't find anything better)
janpeuker-google-sea

Allen Day
thumb

### PR DESCRIPTION
**Pull request by naokichau**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6641,6 +6641,119 @@ <h1>Speakers</h1>
                         </div>
                     </div>
 
+                    <div class="col-md-3 col-sm-6 speaker-column">
+                        <div class="speaker">
+                            <div class="image-holder">
+                                <img class="background-image" alt="Tarun Kumar" src="img/KinKo.jpg">
```
Comment (simsausaurabh) [inline index.html]: ```alt``` value is incorrect.

File: index.html [hunk 2]
```diff
@@ -6641,6 +6641,119 @@ <h1>Speakers</h1>
...
+                            </div>
+                            <span class="speaker-name">Kin Ko</span>
+                            <span>Co-Founder</span>
+                            <span>LikeCoin Foundation</span>
+                        </div>
+                    </div>
+                    <div class="col-md-3 col-sm-6 speaker-column">
+                        <div class="speaker">
+                            <div class="image-holder">
+                                <img class="background-image" alt="Tarun Kumar" src="img/SamWitteveen.jpg">
```
Comment (simsausaurabh) [inline index.html]: ```alt``` value is incorrect.

File: index.html [hunk 3]
```diff
@@ -6641,6 +6641,119 @@ <h1>Speakers</h1>
...
+                            </div>
+                            <span class="speaker-name">Sam Witteveen</span>
+                            <span>Machine Learning Developer Expert</span>
+                            <span>Google</span>
+                        </div>
+                    </div>
+                    <div class="col-md-3 col-sm-6 speaker-column">
+                        <div class="speaker">
+                            <div class="image-holder">
+                                <img class="background-image" alt="Tarun Kumar" src="img/JanPeuker.jpg">
```
Comment (simsausaurabh) [inline index.html]: ```alt``` value is incorrect.

File: index.html [hunk 4]
```diff
@@ -6641,6 +6641,119 @@ <h1>Speakers</h1>
...
+                            </div>
+                            <span class="speaker-name">Jan Peuker</span>
+                            <span>Product Technology Manager</span>
+                            <span>Google</span>
+                        </div>
+                    </div>
+                    <div class="col-md-3 col-sm-6 speaker-column">
+                        <div class="speaker">
+                            <div class="image-holder">
+                                <img class="background-image" alt="Tarun Kumar" src="img/AllenDay.jpg">
```
Comment (simsausaurabh) [inline index.html]: ```alt``` value is incorrect.

File: index.html [hunk 5]
```diff
@@ -6641,6 +6641,119 @@ <h1>Speakers</h1>
...
+                            <span class="speaker-name">Allen Day</span>
+                            <span>Science Advocate</span>
+                            <span>Google</span>
+                        </div>
+                    </div>
+
+                    <div class="col-md-3 col-sm-6 speaker-column">
+                        <div class="speaker">
+                            <div class="image-holder">
+                                <img class="background-image" alt="Tarun Kumar" src="img/VladoKoljibabic.jpg">
```
Comment (simsausaurabh) [inline index.html]: ```alt``` value is incorrect.

**Comment by naokichau** [comment]
oh, sorry, forgot to change that. I will fix soon thanks @simsausaurabh

**Comment by naokichau** [comment]
resolved #492
Preview: https://naokichau.github.io/2018.fossasia.org//

**Comment by mariobehling** [comment]
Thank you!

**Pull request merged**